src/log/normal.c

```diff
@@ -107,30 +107,31 @@ void normal_log_post_test(struct criterion_test_stats *stats) {
             = stats->test_status == CR_STATUS_FAILED ? CRITERION_PREFIX_FAIL :
                                              CRITERION_PREFIX_PASS;
 
-    criterion_plog(level, prefix, _(format),
-            stats->test->category,
-            stats->test->name,
-            stats->elapsed_time);
+    if(stats->test_status == CR_STATUS_SKIPPED) {
+        if(!stats->message) {
+            criterion_pinfo(CRITERION_PREFIX_SKIP, _(msg_post_test_skip),
+                    stats->test->category,
+                    stats->test->name);
+        } else {
+            criterion_pinfo(CRITERION_PREFIX_SKIP, "%s::%s: %s\n",
```

Comment:
I just realized but the format string needs internationalization, perhaps use `_(msg_test_generic)` and rename both definitions of `msg_test_abort` to `msg_test_generic`.